src/locales/index.js

```diff
@@ -18,20 +18,70 @@ const mobileTranslations = {
   es: esMobile
 }
 
-const unsupportedLocaleTag = () => {
+const defaultLanguage = () => {
+  const polyglot = new Polyglot({locale: defaultLocaleTag, phrases: availableTransations[defaultLocaleTag] , onMissingKey: () => null})
+  if (!isDesktop) polyglot.extend(mobileTranslations[defaultLocaleTag])
+  return polyglot
+}
+
+
+const extendPolyglot = (locale, polyglot, phrases, mobilePhrases) => {
+  polyglot.locale(locale)
+  polyglot.extend(phrases)
+  if (!isDesktop) { polyglot.extend(mobilePhrases) }
+  return polyglot
+}
+
+const findMissingKeys = (defaultKeys, customKeys) => {
+  const newTransaltionsSet = new Set(customKeys)
+  const missingKeys =  defaultKeys.filter(element => !newTransaltionsSet.has(element))
+  if (missingKeys.length) { console.warn('Missing keys:', missingKeys) }
+}
+
+const flattenKeys = (phrases, prefix = '') => {
+  return Object.keys(phrases).reduce((result, key) => {
+    if (Array.isArray(phrases[key]) ) {
+      return result
+    }
+    if (phrases[key] !== null && typeof(phrases[key]) === 'object' ) {
+      return [...result, ...flattenKeys(phrases[key], prefix + key + '.')]
```

Comment:
does it only go two levels deep? Seems like recursion would be the way to go here.

Reply:
This is using recursion and it goes multiple level deep until you cannot add another key to the prefix.